src/main/webapp/recipe.html

```diff
@@ -0,0 +1,279 @@
+<!--The MIT License (MIT)
```

Comment:
Please review indent.

Reply:
Fixed in appropriate PR (when I first added this file), which is above this PR on the PR doc.